basestar-schema/src/main/java/io/basestar/schema/FunctionSchema.java

```diff
@@ -269,6 +270,7 @@ public static String getReplacedDefinition(final String definition, final Map<St
             return definition;
         }
         final String groups = using.keySet().stream()
+                .flatMap(name -> Stream.of(name, "\"" + name + "\""))
```

Comment:
interesting that the order does not matter here (I thought quoted must come before unquoted), but I guess since the first character is different, the greedy matching is ok